src/main/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -27,669 +27,704 @@
-
-		@Parameter(required=true, description="<fully qualified method name " +
-				"(with SDK module prefix followed by '.')>")
-		List<String> methodName;
-	}
-
-	private static void showBriefHelp(JCommander jc, PrintStream out) {
-		Map<String,JCommander> commands = jc.getCommands();
-		out.println("");
-		out.println(MODULE_BUILDER_SH_NAME + " - a developer tool for building and validating KBase modules");
-		out.println("");
-		out.println("usage: "+MODULE_BUILDER_SH_NAME+" <command> [options]");
-		out.println("");
-		out.println("    The available commands are:");
-		for (Map.Entry<String, JCommander> command : commands.entrySet()) {
-			out.println("        " + command.getKey() +" - "+jc.getCommandDescription(command.getKey()));
-		}
-		out.println("");
-		out.println("    For help on a specific command, see \"kb-sdk help <command>\".");
-		out.println("    For full usage information, see \"kb-sdk help -a\".");
-		out.println("");
-	};
-
-	private static void showCommandUsage(JCommander jc, HelpCommandArgs helpArgs, PrintStream out) {
-		if(helpArgs.showAll) {
-			showFullUsage(jc, out);
-		} else if(helpArgs.command !=null && helpArgs.command.size()>0) {
-			String indent = "";
-			StringBuilder usage = new StringBuilder();
-			for(String cmd:helpArgs.command) {
-				if(jc.getCommands().containsKey(cmd)) {
-					usage.append(MODULE_BUILDER_SH_NAME + " "+cmd+"\n");
-					jc.usage(cmd,usage,indent+"    ");
-					usage.append("\n");
-				} else {
-					out.println("Command \""+cmd+"\" is not a valid command.  To view available commands, run:");
-					out.println("    "+MODULE_BUILDER_SH_NAME+" "+HELP_COMMAND);
-					return;
+	
+	@Command(
+			name = "run",
+			description = "Run a method of an SDK module registered in the KBase catalog locally."
+	)
+	public static class RunCommand extends TagAndVerbose implements Callable<Integer> {
+		
+		@ArgGroup(exclusive = true, multiplicity = "1")
+		final RunInput input = new RunInput();
+		
+		@Option(
+				paramLabel = "<output_file>",
+				names = {"-o", "--output"},
+				description = """
+						The file where output should be written. If unspecified, output is \
+						written to STDOUT.\
+						"""
+		)
+		Path output;
+		
+		@Option(
+				names = {"-k", "--keep-tmp"},
+				description = "Keep temporary files from the run rather than deleting them.",
+				defaultValue = "false"
+		)
+		boolean keepTempFiles;
+		
+		@Option(
+				paramLabel = "<sdk_home_dir>",
+				names = {"-h", "--sdk-home"},
+				description = """
+						The folder containing the sdk.cfg and run_local folders. If they do \
+						not exist, they will be created. The default is loaded from the \
+						KB_SDK_HOME environment variable, which must be set if this \
+						argument is not supplied.\
+						"""
+		)
+		Path sdkHome;
+		
+		@Option(
+				paramLabel = "<provenance_references>",
+				names = {"-r","--prov-refs"},
+				description = """
+						A comma separated list of KBase workspace object addresses in the format \
```

Comment:
comma-separated

Reply:
Fixed locally, will show up in a later PR